packages/cli/src/services/config.service.ts

```diff
@@ -0,0 +1,23 @@
+import * as Configstore from 'configstore';
+import jwt_decode from 'jwt-decode';
+import { IJwt } from '../interfaces';
+
+export class ConfigService {
+  _config: Configstore;
```

Comment:
If this private, you can further guard against it with typescript:
```suggestion
  private _config: Configstore;
```